pos_cashier_login/static/src/js/main.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2019 by Lambda IS <https://www.lambda-is.com/>
+
+odoo.define('pos_cashier_login.main', function (require) {
+    'use strict';
+
+    var gui = require('point_of_sale.gui');
+    var screens = require('point_of_sale.screens');
+    var chrome = require('point_of_sale.chrome');
+    var pos_models = require('point_of_sale.models');
+    var SuperPosModel = pos_models.PosModel.prototype;
+
+    pos_models.PosModel = pos_models.PosModel.extend({
```

Comment:
you may use an include instead:
```js
pos_models.PosModel.include({
...
})
```

Reply:
Tried but it is a backbone model and it doesn't have such a method.

Reply:
ok :)